src/UFS2ARCO/ufsdataset.py

```diff
@@ -216,7 +218,7 @@ def _store_coordinates(self, cds):
         cds.to_zarr(store, mode="w")
         print(f"Stored coordinate dataset at {self.coords_path}")
 
-    def _store_data_vars(self, xds, **kwargs):
+    def _store_data_vars(self, xds: xr.Dataset, **kwargs) -> None:
```

Comment:
Thanks for all of these type annotations!

Reply:
Now you are just being sarcastic! ;)